br/pkg/storage/azblob.go

```diff
@@ -77,13 +77,17 @@ func defineAzblobFlags(flags *pflag.FlagSet) {
 	flags.String(azblobAccountKey, "", "Specify the account key for azblob")
 	flags.String(azblobSASToken, "", "Specify the SAS (shared access signatures) for azblob")
 	flags.String(azblobEncryptionScope, "", "Specify the server side encryption scope")
+
+	_ = flags.MarkHidden(azblobEncryptionScope)
 }
 
 func hiddenAzblobFlags(flags *pflag.FlagSet) {
 	_ = flags.MarkHidden(azblobEndpointOption)
 	_ = flags.MarkHidden(azblobAccessTierOption)
 	_ = flags.MarkHidden(azblobAccountName)
 	_ = flags.MarkHidden(azblobAccountKey)
+	_ = flags.MarkHidden(azblobSASToken)
+	//_ = flags.MarkHidden(azblobEncryptionScope)
```

Comment:
Please add a comment about why this should be moved out...

Reply:
remove the comment, just mark hidden twice